### ISSUE TITLE
Mobile Navigation

### ISSUE THREAD

**Issue by garingx** [ninacoder-info/music-engine-web-wiki#31]
While I love all the mobile navigation features. The only thing that keeps confusing me is the back button. Personally, I would rather have the burger menu stuck there at all times.

**Comment by dabalalakuise** [comment]
You are right. The theme needs a lot of improvement. The burger menu stuck there would decrease bounce rate
